### DISCUSSION TITLE
Whether there are plans for risc-v architecture support？

### DISCUSSION THREAD

**Discussion by wzzc-dev** [oracle/graal#8657] [General]
I know there are ways to build with llvm, do I know if there is a way to run directly on risc-v architecture machines

**Comment by alina-yur** [comment]
hi @wang-zhichuang, at the moment building via the LLVM backend (https://medium.com/graalvm/graalvm-native-image-meets-risc-v-899be38eddd9) is the best way to go.

**Reply by f123888** [reply]
But it looks like a experiment way. I don't find any performance and accuracy test report. And there is no release binary on official website.
Will it really be put into production?
